AirCasting/Map/PathPoint.swift

```diff
@@ -18,9 +18,9 @@ struct PathPoint: Equatable {
     
     let location: CLLocationCoordinate2D
     let measurementTime: Date
-    let measurement: Double
+    var measurement: Double = 20
```

Comment:
that's not resolving the issue - it changes its location from being in multiple places to being in a single place which **great**, but this is still a code smell that indicates we have a wrong abstraction here perhaps. Why is `googleLocation` using PathPoint? It is is basically a measurement with added location, maybe we need two objects instead of one?

Please keep the warning and add an issue to github accordingly

Reply:
Yeah, right. I am going to add now